schedule/yast/zfcp.yaml

```diff
@@ -47,44 +51,44 @@ test_data:
         installed: 1
   zfcp:
     fcp_devices:
-      - fcp_channel: '0.0.fa00'
```

Comment:
what is the issue to change to double quotes? Seems fine, even when there are no spaces is not required to use any quotes in yaml.